src/ott/neural/flows/flows.py

```diff
@@ -0,0 +1,157 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import abc
+
+import jax.numpy as jnp
+
+__all__ = [
+    "BaseFlow",
+    "StraightFlow",
+    "ConstantNoiseFlow",
+    "BrownianNoiseFlow",
+]
+
+
+class BaseFlow(abc.ABC):
+  """Base class for all flows.
+
+  Args:
+    sigma: Constant noise used for computing time-dependent noise schedule.
+  """
+
+  def __init__(self, sigma: float):
+    self.sigma = sigma
+
+  @abc.abstractmethod
+  def compute_mu_t(
+      self, t: jnp.ndarray, src: jnp.ndarray, tgt: jnp.ndarray
+  ) -> jnp.ndarray:
+    """Compute the mean of the probablitiy path.
+
+    Compute the mean of the probablitiy path between :math:`x` and :math:`y`
```

Comment:
For consistency with the rest of the docstrings, use :math\`x_0\` and :math:\`x_1\`.